### DISCUSSION TITLE
Add support for local Jackett

### DISCUSSION THREAD

**Discussion by Xazmah** [Purfview/IMDb-Scout-Mod#64] [Ideas]
Most added indexers in Jackett support searching by id, URLs are usually like this

http://localhost:9117/UI/Dashboard#search=tt0055630

**Comment by Purfview** [comment]
What is the point to search same thing twice?

**Comment by Xazmah** [comment]
Maybe added to Icon sites (no search) section, cause I have auto search disabled, I don't want to auto search every title page I visit

**Comment by Purfview** [comment]
That link opens Jackett's dashboard?
I'll look where it fits, maybe at the special buttons section (they still are added at the end of the icon sites).

**Comment by Purfview** [comment]
Or I got better idea, how about button to start searching sites? At the end of the search it would turn auto-search setting off again.

**Reply by Xazmah** [reply]
Yeah great idea if you can implement it

**Comment by Xazmah** [comment]
Yes it does open the Jackett dashboard and start searching all Indexers you added, not every indexer support searching by id though, So I guess 3 icons are needed one by id, one by string + year for movies and another one with just string for tv shows

**Comment by Xazmah** [comment]
The icon is usually http://localhost:9117/favicon.ico

**Comment by Purfview** [comment]
> Yeah great idea if you can implement it

But wait, there is kinda similar function, uncheck "Load links to sites on start" in settings and you will have auto search on button.

> Yes it does open the Jackett dashboard and start searching all Indexers you added, not every indexer support searching by id though, So I guess 3 icons are needed one by id, one by string + year for movies and another one with just string for tv shows

That's a lot of work for ~duplicate search...

Anyway, I'll add these ideas to the bottom of my list, so expect it later than sooner, if ever. :)

**Comment by Xazmah** [comment]
> But wait, there is kinda similar function, uncheck "Load links to sites on start" in settings and you will have auto search on button.

First time noticing it's existence 

> That's a lot of work for ~duplicate search...

Only search by id is important for me since most trackers support it 

> Anyway, I'll add these ideas to the bottom of my list, so expect it later than sooner, if ever. :)

I wanted to submit a PR but didn't know in which section to put Jackett

**Reply by Purfview** [reply]
> I wanted to submit a PR but didn't know in which section to put Jackett.

Does it check for results and if Jackett is running? What it does if Jackett is not installed?
If it's just a link, then keep it at your fork for now.

Btw, can you have your modifications at your fork and import only changes from the master repo? I'm not sure how this all works, but I think it should work like that. That way you could set userscript to update from your own fork, so you wouldn't need to re-copy-paste your custom sites after every update.

**Reply by Xazmah** [reply]
It's just an icon to open Jackett search in a new tab (no search section), why would anyone not having Jackett installed have it activated, I'll try add it to my fork
